src/Database/DBStructure.php

```diff
@@ -1716,7 +1716,6 @@ public static function definition() {
 						"created" => ["type" => "datetime", "not null" => "1", "default" => NULL_DATE, "comment" => ""],
 						"received" => ["type" => "datetime", "not null" => "1", "default" => NULL_DATE, "comment" => ""],
 						"global" => ["type" => "boolean", "not null" => "1", "default" => "0", "comment" => ""],
-						"aid" => ["type" => "int unsigned", "not null" => "1", "default" => "0", "comment" => ""],
```

Comment:
No database version change?

Reply:
This isn't needed, since we don't have to execute an updater command. This field had never been used,